images/jupyter/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
```

Comment:
Please remove this and add make command for `capella-docker-images` in the Makefile.